hoodie-client/src/main/java/com/uber/hoodie/index/bloom/BloomIndexFileInfo.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *  Copyright (c) 2017 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package com.uber.hoodie.index.bloom;
+
+import com.google.common.base.Objects;
+
+import java.io.Serializable;
+
+/**
+ * Metadata about a given file, useful for index lookup
+ */
+public class BloomIndexFileInfo implements Serializable {
+
+    private final String fileName;
+
+    private final String minRecordKey;
```

Comment:
they are null, using a different constructor. Optional's original intention is to be used for return types.